Tests/Gedmo/Form/TranslationsTypeTest.php

```diff
@@ -18,6 +18,16 @@ protected function getUsedEntityFixtures()
 
     public function testSubmitValidDefaultConfigurationData()
     {
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $formType = 'Symfony\Component\Form\Extension\Core\Type\FormType';

```

Comment:
IMHO, we can safely bump to PHP 5.6 minimum support and use class keywords.

We can also bump to Symfony 2.8 as minimum and use Class type directly.

Reply:
I have dropped 5.3 because I cannot manage it to work with it, and travis complains a lot. But I would like to setup php-cs-fixer and flintCI and scrutinizer, before moving to php5.6 and Symfony 2.8

wdyt?

Reply:
And probably add test for the dependency injection first too